CHANGELOG.md

```diff
@@ -22,7 +22,46 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Known Issues 🚫
 
 - ReadOnly attribute in PVC template not fully handled
-- Volume restore/clone not implemented
+
+## [v0.12.0] - 2025-11-20 ⚠️ Breaking Changes
+
+### Added ✨
+
+- Volumes Snapshots using the rawfile image itself
+  - If the underlying fs supports it, we use `reflinks` for `COW`
+  - Otherwise, a deep copy is performed and application is frozen with `fsfreeze`
```

Comment:
Here we should mention that we are blocking snapshot create of in use volumes when fsfreeze is not enabled in storage class params